src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -439,15 +442,18 @@ public static class TextCatalog
                 useAllLengths: useAllLengths, numberOfBits: numberOfBits, seed: seed, useOrderedHashing: useOrderedHashing, maximumNumberOfInverts: maximumNumberOfInverts, rehashUnigrams) });
 
         /// <summary>
-        /// Produces a bag of counts of hashed ngrams in <paramref name="inputColumnNames"/>
-        /// and outputs ngram vector as <paramref name="outputColumnName"/>
-        ///
+        /// Create a <see cref="NgramHashingEstimator"/>, which copies the data from the columns specified in <paramref name="inputColumnNames"/>
```

Comment:
>the columns [](start = 85, length = 11)

columns -> multiple columns

let's be explicit. in the first read of the summary, it's not clear how the two overloads are different.